### DISCUSSION TITLE
Welcome to nuggies-npm Discussions!

### DISCUSSION THREAD

**Discussion by Assassin-1234** [Nuggies-bot/nuggies-npm#18] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by face-hh** [comment]
ASS

**Reply by Assassin-1234** [reply]
hi

**Reply by face-hh** [reply]
@HypsterOP#5687 discord.js-helper

**Reply by face-hh** [reply]
hi

**Comment by face-hh** [comment]
ASS

**Reply by Assassin-1234** [reply]
hi

**Reply by face-hh** [reply]
@HypsterOP#5687 discord.js-helper

**Reply by face-hh** [reply]
hi
